src/NuGet.Core/NuGet.Packaging/NuspecReader.cs

```diff
@@ -113,6 +113,14 @@ public IEnumerable<PackageDependencyGroup> GetDependencyGroups()
         /// Read package dependencies for all frameworks
         /// </summary>
         public IEnumerable<PackageDependencyGroup> GetDependencyGroups(bool useStrictVersionCheck)
+        {
+            return GetDependencyGroups(useStrictVersionCheck: useStrictVersionCheck, allowTokens: false);
+        }
+
+        /// <summary>
+        /// Read package dependencies for all frameworks
+        /// </summary>
+        public IEnumerable<PackageDependencyGroup> GetDependencyGroups(bool useStrictVersionCheck, bool allowTokens)
```

Comment:
default this to false so as to minimize changes in callers

Reply:
It's already defaulted by way of the other overload, does that work?